luigi/contrib/rdbms.py

```diff
@@ -27,7 +27,125 @@
 logger = logging.getLogger('luigi-interface')
 
 
-class CopyToTable(luigi.task.MixinNaiveBulkComplete, luigi.Task):
+class _MetadataColumnsMixin(object):
+    """Provide an additinal behavior that adds columns and values to tables
+
+    This mixin is used to provide an additional behavior that allow a task to
+    add generic metadata columns to every table created for both PSQL and
+    Redshift.
+
+    Example:
+
+        This is a use example or how this mixin could come handy and how to use
```

Comment:
Are you trying to say `This is an example use-case of how this mixin could come in handy.`?